docs/src/index.md

```diff
@@ -496,26 +473,33 @@ x_thread = @chain df begin
 end
 ```
 
-Another alternative is Pipe.jl which exports the `@pipe` macro for piping. 
-The piping mechanism in Pipe requires explicit specification of the piped
-object via `_` instead of assuming it is the first argument to the next function.
-The Pipe.jl equivalent of the above is:
+By default, `@chain` places the argument returned from the 
```

Comment:
```suggestion
By default, `@chain` places the value of the 
```
(expression is by definition something that has a value; I expecially like this explanation https://fsharpforfunandprofit.com/posts/expressions-vs-statements/#:~:text=In%20programming%20language%20terminology%2C%20an,and%20doesn't%20return%20anything.)